Google.PowerShell/Provider/GoogleCloudStorageProvider.cs

```diff
@@ -835,6 +836,7 @@ protected override void RemoveItem(string path, bool recurse)
                 case GcsPath.GcsPathType.Object:
                     if (IsItemContainer(path))
                     {
+                        path = path.TrimEnd("/\\".ToCharArray());
```

Comment:
String.TrimEnd can take a params array. This could be `path.TrimEnd('/', '\\');`